feat: add product detail table

Aadarsh-shrivastava opened the pull request:
add product detail table and fix issue with responsive layout

![Screenshot 2025-03-12 153536](https://github.com/user-attachments/assets/406b4bce-a2b9-4bfc-a08c-29a66ecb14ec)
![Screenshot 2025-03-12 153527](https://github.com/user-attachments/assets/4ce12e6d-e5ff-4246-a3b7-f54fb163aa11)
![Screenshot 2025-03-12 151758](https://github.com/user-attachments/assets/8aa002f1-ca17-4485-bcff-709096179423)
![Screenshot 2025-03-12 151731](https://github.com/user-attachments/assets/dcf16168-bd90-4adc-bb55-e822da67be6f)

Aadarsh-shrivastava committed:
feat: add product detail table
fix: fix issue with responsive layout

shah-harshit reviewed (commented):
Rename space_dashboard (2).svg

Aadarsh-shrivastava committed:
chore: rename asset
debug: fix sidebar background blur behaviour

Aadarsh-shrivastava committed:
refactor: css code for readability
feat: dark mode and light mode switch on table and grid icons
debug: fix drop down not staying open issue

shah-harshit commented on index.html:
```diff
@@ -577,7 +618,248 @@
+            <p class="card-title">Product Details</p>
+            <div class="data-switch-button-group">
+              <label for="showTable" id="show-table-label">
+                <!-- <img class="icon-mid" src="./assets/table_dark.svg" /> -->
```
Remove commented code.

shah-harshit commented on index.html:
```diff
@@ -577,7 +618,248 @@
+                <!-- <img class="icon-mid" src="./assets/table_dark.svg" /> -->
+              </label>
+              <label for="showGrid" id="show-grid-label">
+                <!-- <img class="icon-mid" src="./assets/grid_dark.svg" /> -->
```
Remove commented code.

Aadarsh-shrivastava committed:
refactor: removed commented code
